README.md

```diff
@@ -92,6 +107,84 @@ vars:
 ```
 The corresponding metrics from the disabled tables will not populate in the downstream models.
 
+## Optional: Utilizing Salesforce History Mode records
```

Comment:
Please see my source package review notes for similar updates that should be applied to this README as well.

Reply:
Updated